src/csync/csync_exclude.h

```diff
@@ -29,6 +29,7 @@
 #include <QSet>
 #include <QString>
 #include <QRegularExpression>
+#include <QFile>
```

Comment:
please forward declare `QFile` and only add the include in the `.cpp` file
that will help decreasing compilation time and decouple the implementation and the interface a bit more